test/runtests.jl

```diff
@@ -722,7 +721,7 @@ end
 end
 
 ## https://github.com/JuliaDiff/ChainRules.jl/tree/master/test/rulesets
-if !Sys.iswindows()
+if !Sys.iswindows() && !(Sys.ARCH == :aarch64 && Sys.isapple())
```

Comment:
is this needed?

Reply:
This test suite is currently broken on M1/M2 with an assert failure in Enzyme proper. Will remove for now.